components/authz-service/server/v2/project_update_manager.go

```diff
@@ -432,3 +404,45 @@ func (m *CerealProjectUpdateManager) Status() (ProjectUpdateStatus, error) {
 	}
 	return projectUpdateInstance, nil
 }
+
+// Find the longest estimate that is in the future. If no estimate is found return zero time.Time{}
+func findLongestEstimatedTime(estimatedTimes []time.Time) time.Time {
+	longestEstimatedTimeComplete := time.Time{}
+	for _, estimatedTime := range estimatedTimes {
+		if estimatedTime.After(time.Now()) && estimatedTime.After(longestEstimatedTimeComplete) {
```

Comment:
It's probably inconsequential, but `time.Now()` will be different for every iteration through this loop. Maybe we should define `noew := time.Now()` before that loop and compare to this here? 💭